examples/ads.amp.html

```diff
@@ -401,13 +401,25 @@ <h2 id="contentad">Content.ad Banner 320x50</h2>
     <div fallback></div>
   </amp-ad>
 
-  <h2 id="criteo">Criteo (0x0 fake ad)</h2>
+  <h2 id="criteo">Criteo Passback</h2>
   <amp-ad width="300" height="250"
       type="criteo"
+      data-tagtype="passback"
```

Comment:
Yes, it is probably related to targeting (and the same will probably happen for the RTA one). 

Could we leave it as is for now?